docs/docsite/rst/dev_guide/style_guide/preferred_terms.rst

```diff
@@ -0,0 +1,58 @@
+.. _preferred_terms:
+
+Preferred terminology
+=====================
+
+.. contents::
+  :local:
+
+The following is the list of preferred terms for Ansible ecosystem projects.
+
+.. glossary::
+
+   Ansible
+   Use on all occurrences or ``ansible`` if referring to the CLI command. Preference is to use this to refer to the Ansible community package.
+Ansible Builder
```

Comment:
May I suggest injecting empty line separators before each term? It makes a huge visual difference when looking at the source code.
```suggestion

Ansible Builder
```

Reply:
@samccann also, my original suggested change only indented the first term but all the others should be indented too, it just wasn't convenient to do this in a browser input.